content/management/configuration-api/v3.2/index.mdx

```diff
@@ -32,6 +32,236 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
+| --------------- | ------------ | --------- | ----------------------------- |
+| `access`        | Yes          | `object`  | Destination access            |
+| `access.groups` | Yes          | `[]int`   | Destination groups            |
+| `conditions`__**__   | Yes          | `object`  | Conditions to check                |
+| `conditions.customer_url` | No      | `object`  | The condition that matches on `customer_page.url` from **Login** or `url` from **Update Customer Page** |
+| `conditions.customer_domain` | No   | `object`  | The condition that matches on the domain of the `customer_url` |
+| `conditions.customer_*.values` | No | `[]match_object`_*_ | Positive match, may not be used together with `exclude_values` |
+| `conditions.customer_*.exclude_values` | No | `[]match_object` | Negative match |
+| `conditions.geolocation` | No   | `object`  | The condition that matches on the customer's geolocation |
+| `conditions.geolocation.values` | Yes | `[]geolocation_object`__**__ | An array of objects; matches when at least one subcondition matches. |
+| `description`   | No           | `string`  | Description of the auto access  |
+
+__*__ `<match_object>` structure:
+
+| Parameter   | Required | Data type | Notes |
+| ----------- | -------- | --------- | ----- |
+| `value`     | Yes      | `string`  | A value to match |
+| `exact_match` | No     | `bool`    | If set to `false`, a `match_object.value` will be matched as a substring of the customer URL or domain. |
+
+__**__ the `geolocation_object` structure.  The condition is fulfilled only when all of the specified fields match:
+
+| Parameter   | Required | Data type | Example |
+| ----------- | -------- | --------- | ----- |
+| `country`   | No | `string` | "United States |
+| `country_code` | No | `string` | "US" |
+| `region` | No | `string` | "California" |
+| `city` | No | `string` | "Mountain View" |
+
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/add_auto_access \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+        "description": "Chats on livechat.com from United States",
+        "access": {
+          "groups": [ 1 ]
+        },
+        "conditions": {
+          "customer_domain": {
+            "values": [
+              {
+                "value": "livechat.com",
+                "exact_match": true
+              }
+            ]
+          },
+          "customer_geolocation": {
+            "values": [
+              {
+                "country": "United States",
+                "country_code": "US"
+              }
+            ]
+          }
+        }
+      }'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+  "id": "pqi8oasdjahuakndw9nsad9na"
+}
+```
+
+</CodeResponse>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Get auto access config
```

Comment:
```suggestion
### List Auto Accesses
```

Reply:
+ a note in Changelog:

There are new methods for managing auto access: [**Add Auto Access**](/management/configuration-api/v3.2/#add-auto-access),  [**List Auto Accesses**](/management/configuration-api/v3.2/#list-auto-accesses), and [**Delete Auto Access**](/management/configuration-api/v3.2/#delete-auto-access).